e2e/package.json

```diff
@@ -8,6 +8,7 @@
     "preview": "vite preview",
     "serve": "serve lib --listen 4173 -L",
     "test": "playwright test",
+    "test:debug": "playwright test --ui",
```

Comment:
This is mentioned in https://github.com/Milkdown/milkdown/blob/main/CONTRIBUTING.md